eventmesh-server-go/runtime/core/protocol/grpc/consumer/consumer_group_option_test.go

```diff
@@ -0,0 +1,22 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package consumer
+
+import "testing"
+
+func Test_NewConsumerGroupTopicOption(t *testing.T) {
+
```

Comment:
There is no testcode

Reply:
Modify the project structure first, the original structure is not easy to write mocks code, in order to prevent too much code submitted at one time, so submit this pr first.